addons/mail/static/src/model/tests/model_field_command_tests.js

```diff
@@ -0,0 +1,714 @@
+/** @odoo-module **/
+
+import {
+    clear,
+    create,
+    decrement,
+    increment,
+    insert,
+    insertAndReplace,
+    link,
+    replace,
+    unlink,
+    unlinkAll,
```

Comment:
This file is getting quite big already, I would suggest to split and make 1 file per command.